doc/ideas/gameplay.md

```diff
@@ -18,37 +18,45 @@ Environment
 
 ### Fire
 
- - Buildings made out of wood and other flammable material can be lit.
- - Fire departments in the city can extinguish those fires (Stronghold anybody?).
+ - Buildings made out of wood and other flammable material can be lit
+ - Fire departments in the city can extinguish those fires (Stronghold anybody?)
  - Sneaky spies could burn your city down
 
 ### Day/night cycle
 
- - At night: View distance reduced, everything darker.
- - Villagers need sleep, otherwise they die at work.
+ - At night: View distance reduced, everything darker
+ - Villagers need sleep, otherwise they die at work
  - Chemistry industry: Coffeine, Peniciline, ...?
 
+### Forests
+
+ - A single villager can hide (per some tile area) in the forest for sneak attacks. Maybe can only visible to a certain type of unit. But invisible to other villagers (until they pop out and start building stuff)
+
+### Forests
+
+ - A single villager can hide (per some tile area) in the forest for sneak attacks. Maybe can only visible to a certain type of unit. But invisible to other villagers (until they pop out and start building stuff)
+
```

Comment:
Now this section is duplicated.

Reply:
Damn, really appreciate the patience. Let me try that one more time.